openstack/nova-hypervisor-agents/templates/hypervisors-kvm-daemonset.yaml

```diff
@@ -111,23 +111,7 @@ spec:
           command:
           - "nova-compute"
           env:
-            - name: OS_LIBVIRT__LIVE_MIGRATION_INBOUND_ADDR
-              valueFrom:
-                fieldRef:
-                  fieldPath: status.hostIP
-            - name: OS_VNC__SERVER_LISTEN
-              valueFrom:
-                fieldRef:
-                  fieldPath: status.hostIP
-            - name: OS_VNC__SERVER_PROXYCLIENT_ADDRESS
-              valueFrom:
-                fieldRef:
-                  fieldPath: status.hostIP
-            - name: OS_SPICE__SERVER_LISTEN
-              valueFrom:
-                fieldRef:
-                  fieldPath: status.hostIP
-            - name: OS_SPICE__SERVER_PROXYCLIENT_ADDRESS
+            - name: OS_DEFAULT__MY_IP
```

Comment:
Also unrelated to serial.

Reply:
It is related, since `proxyclient_address` is supposed to use this value.